sections/parties.md

```diff
@@ -24,6 +24,15 @@ Results are paged in groups of 25. Use `?n=25` to check for the next 25 results
 </parties>
 ```
 
+Strict party search
+-------------------
+
+Add `&strict=1` after the query parameters to return strict name results. 
+
+For example: `/parties/search.xml?name=jon&strict=1` returns any contacts with the name `Jon`. 
```

Comment:
This makes sense I think. There's also the behavior with the last name, but those should be exact as well so maybe that's overkill. The caveat with only first names is that it'll return `Jon`, `Jon Phenow`, `Jon Michael` - so it matches on all first names.

Reply:
See description here for a little more bullet-point description of it https://github.com/highrisehq/server/pull/2178